### PR TITLE
Illustrations

### PR DESCRIPTION
**Pull request by mviegasb**
Have had a first stab at the updated images page. Let me know thoughts / what needs changing.

### CODE REVIEW

File: app/views/styles/images.html [hunk 1]
```diff
@@ -15,12 +15,53 @@
...
+        <h2 class="govuk-heading-l">Do you have biometric residence card or permit?</h2>
+        <img src="{{ asset_path }}/images/image-brp-example2.png" class="confirm-desktop" alt="An example of an illustrative image of a biometric residence permit card">
+        <img src="{{ asset_path }}/images/image-brp-example2.png" class="confirmation confirm-mobile" alt="An example of an illustrative image of a biometric residence permit card">
+        <span class="image-caption"><a href="www.right-to-work.service.gov.uk">Right to work</a></span>
+      </div>
+
+      <div class="example example-images confirmation" style="margin-bottom: 2em;">
+        <img src="{{ asset_path }}/images/image-passport-issue-example.png" class="confirm-desktop" alt="An example of an illustrative image of a passport and signposting where to find it's issue date.">
+        <img src="{{ asset_path }}/images/image-passport-issue-example.png" class="confirmation confirm-mobile" alt="An example of an illustrative image of a passport and signposting where to find it's issue date.">
+        <span class="image-caption"><a href="https://www.gov.uk/apply-renew-passport">Passport renewals</a></span>
```
Comment (karypun) [inline app/views/styles/images.html:50]: Broken image

File: app/views/styles/images.html [hunk 2]
```diff
@@ -15,12 +15,53 @@
...
+
+      <h2 id="Illustrations or representative imagery" class="govuk-heading-l">Illustrations or representative imagery</h2>
+      <p class="govuk-body">If your image represents something physical, such as a document, you should use the aspect ratio of that object.</p>
+      <p class="govuk-body">Illustrations must have a consistent style to create trust across online and print.</p>
+
+      <div class="example example-images confirmation" style="margin-bottom: 2em;">
+        <h2 class="govuk-heading-l">Do you have biometric residence card or permit?</h2>
+        <img src="{{ asset_path }}/images/image-brp-example2.png" class="confirm-desktop" alt="An example of an illustrative image of a biometric residence permit card">
+        <img src="{{ asset_path }}/images/image-brp-example2.png" class="confirmation confirm-mobile" alt="An example of an illustrative image of a biometric residence permit card">
+        <span class="image-caption"><a href="www.right-to-work.service.gov.uk">Right to work</a></span>
```
Comment (karypun) [inline app/views/styles/images.html:44]: Right to work link broken

File: app/views/styles/images.html [hunk 3]
```diff
@@ -32,55 +73,44 @@ <h2 class="govuk-heading-m">Use of images</h2>
...
+            </div>
+            <div class="govuk-grid-column-one-half">
+              <span class="govuk-heading-m">Don't</span>
+              <img src="{{ asset_path }}/images/image-example-1.jpg" alt="Two people talking at work. Image has a purple filter." class="image-examples">
+              <span class="image-caption">Don't use filters or effects</span>
+            </div>
           </div>
         </div>
+
+      <h2 id="Alternative text" class="govuk-heading-l">Alternative text</h2>
```
Comment (karypun) [inline app/views/styles/images.html:108]: Add a break before Alternative text heading (h2 sits right underneath example box.)

File: app/views/styles/images.html [hunk 4]
```diff
@@ -15,12 +15,53 @@
...
+        <li>a new or unfamiliar concept</li>
+        <li>complex guidance when completing a task</li>
+        <li>how to find a reference number on a document complex.</li>
+      </ul>
+
+      <p class="govuk-body"><a href="#Photography">Photography</a> can add visual interest to a page. We use it mostly in internal pages and services that use independent branding.</p>
+      <p class="govuk-body">Images must be relevant to the content, help users understand what they need to do and follow our guidance on the use of <a href="#Alternative text">Alternative text</a>.</p>
+
+      <h3 class="govuk-heading-m">Sourcing images</h3>
+      <p>Please contact the central design team (<a href="mailto:designops@digital.homeoffice.gov.uk">designops@digital.homeoffice.gov.uk</a>) for advice on image use, for original source files and to commission illustrations.</p>
```
Comment (karypun) [inline app/views/styles/images.html:34]: When we link to designops email address, should we be consistent about who we are? (we're central design team here but design system working group in "Get in touch" blue bar below).

Comment (mviegasb) [reply app/views/styles/images.html]: Have changed this to design system working group in both & changing the email to the design system email (still to be created).

File: app/views/styles/images.html (hunk 3 above)
Comment (mviegasb) [reply app/views/styles/images.html:108]: Have added a <br>, is that the correct way to add a break?

File: app/views/styles/images.html (hunk 2 above)
Comment (mviegasb) [reply app/views/styles/images.html]: Have updated this now, had missed out the https://

File: app/views/styles/images.html (hunk 1 above)
Comment (mviegasb) [reply app/views/styles/images.html]: Not sure why the image is broken? It seems to work on my local machine. I'll take another look on Monday.

File: app/views/styles/images.html (hunk 3 above)
Comment (karypun) [reply app/views/styles/images.html:108]: Yes looks good!

File: app/views/styles/images.html (hunk 1 above)
Comment (mviegasb) [reply app/views/styles/images.html]: I can't see an issue on my local now - think I've fixed this?

**Comment by mviegasb** [comment]
@karypun I've updated the contact email address back to designops now and fixed the broken image.

**Pull request merged**